runtime/onert/core/src/exec/Executors.cc

```diff
@@ -468,10 +545,10 @@ void Executors::execute(const IODescription &desc)
           _edge_tensors[from_iodesc]->decrease_ref();
 
           // Decrease reference count of nnpkg inputs
-          // Assumption: nnpkg inputs/outputs are not duplicated with edges.
-          if (pkg_io_edge_tensors.find(to_iodesc) != pkg_io_edge_tensors.end())
```

Comment:
`pkg_io_edge_tensors` was separated into `_pkg_input_quant_tensors` and `_pkg_input_quant_tensors` in order to avoid duplication of `IODesc` of inputs and outputs.